doc/api/module.md

```diff
@@ -135,26 +135,19 @@ import { findSourceMap, SourceMap } from 'module';
 const { findSourceMap, SourceMap } = require('module');
 ```
 
-### `module.findSourceMap(path[, error])`
+### `module.findSourceMap(path)`
```

Comment:
Not blocking, but this kind of change changes the URL linking to this section, so there's a possibility that there are some links on StackOverflow or on blog articles "breaking" if we change it. In other part of the documentation, we add an invisible anchor with the old ID to make sure old links still work.

We should also change `modules.md`, there's a list of methods linking to here (`make test-doc` should highlight it because that would be a broken link).

Reply:
@bcoe The following line needs to be changed:

https://github.com/nodejs/node/blob/9dbde1d4fc0f5c3acf8c009339b37805b2460fbb/doc/api/modules.md#L970

to this:

```markdown
* <a id="modules_module_findsourcemap_path_error" href="module.html#module_module_findsourcemap_path">`module.findSourceMap(path)`</a>
```

Reply:
Semi unrelated and not for this PR, but I think:

> Not blocking, but this kind of change changes the URL linking to this section, so there's a possibility that there are some links on StackOverflow or on blog articles "breaking" if we change it.

Is an excellent point and something our md linter _should_ probably complain about.

Reply:
I've added an anchor above the `findSourceMap` method 👍